lib/web/apiserver.go

```diff
@@ -1989,10 +1989,10 @@ func (h *Handler) validateTrustedCluster(w http.ResponseWriter, r *http.Request,
 
 	validateResponse, err := h.auth.ValidateTrustedCluster(validateRequest)
 	if err != nil {
+		log.Errorf("failed validating trusted cluster: %v", err)
```

Comment:
log.WithError(err).Error("Failed to validate trusted cluster token.")

Reply:
done